src/in-depth/render-configuration.md

```diff
@@ -0,0 +1,46 @@
+# Rendering documentation for your Configuration
```

Comment:
Could you open a discussion for us to discuss this first?

There are a lot of potential directions to go and I tend to prefer the solution discover to happen in issues as PRs are more likely to come and go.